File: practice_7/p1/code/HW_practice-7_p1.cpp
#include <iostream>
// "Print such an Array first which Elements are Greater Than t" is P.A.E.G.T.t.
void PAEGTt(int* arrA, int* arrB, int t)
{
    int cA = 0;
    int cB = 0;
    for (int i = 0; i !=5; i++)
    {
        if (arrA[i] > t)
        {
            cA+=1;
        }
        if (arrB[i] > t)
        {
            cB+=1;
        }
    }
    if (cB > cA)
    {
        std::cout << "\tМассив B:\n";
        for (int i=0; i!=5; i++)
        {
            std::cout << arrB[i] << ' ';
        }
        std::cout << std::endl;
        
        std::cout << "\tМассив A:\n";
        for (int i=0; i!=5; i++)
        {
            std::cout << arrA[i] << ' ';
        }
        std::cout << std::endl;
    }
    else if (cA > cB)
    {
        std::cout << "\tМассив A:\n";
        for (int i=0; i!=5; i++)
        {
            std::cout << arrA[i] << ' ';
        }
        std::cout << std::endl;;
        
        std::cout << "\tМассив B:\n";
        for (int i=0; i!=5; i++)
        {
            std::cout << arrB[i] << ' ';
        }
        std::cout << std::endl;
    }
    else
    {
        std::cout << "Кол-во таковых элементов в обоих массивах одинаково.\nВывожу оба массива:\n";
        std::cout << "\tМассив A:\n";
        for (int i=0; i!=5; i++)
        {
            std::cout << arrA[i] << ' ';
        }
        std::cout << std::endl;;
        
        std::cout << "\tМассив B:\n";
        for (int i=0; i!=5; i++)
        {
            std::cout << arrB[i] << ' ';
        }
        std::cout << std::endl;
    }
}

int main() {
    int aA[5] = {5, 6, 2, 9, 3};
    int aB[5] = {8, 5, 1, 0, 4};
    PAEGTt(aA, aB, 8);
    return 0;
}

File: practice_7/p1/code/HW_practice-7_p1.py
# "Print such an Array first which Elements are Greater Than t" is P.A.E.G.T.t.
def PAEGTt(arrA, arrB, t):
    cA = 0
    cB = 0
    for i in range(0, 5):
        if arrA[i] > t:
            cA += 1
        if arrB[i] > t:
            cB += 1
    if cB > cA:
        return print(f'\tМассив B: {arrB}' 
                     f'\n\tМассив A: {arrA}')
    elif cA > cB:
        return print(f'\tМассив A: {arrA}'
                     f'\n\tМассив B: {arrB}')
    else:
        return print(f'\nКол-во таковых элементов в обоих массивах одинаково.'
                  f'\nВывожу оба массива:'
                  f'\n\tМассив A:\n  {arrA} \n\tМассив B:\n  {arrB}')

ent_t = int(input('Введите t: '))
A = [66,8,18,3,11]
B = [34,21,87,51,5]
PAEGTt(A,B,ent_t)
